backend/app/schemas.py

```
@@ -119,53 +142,53 @@
-    quantity: int = Field(..., ge=1)  
-   
-  
-class OrderCreate(BaseModel):  
+class OrderItemCreate(BaseModel):
+    product_id: int
+    size: Optional[str]
+    quantity: int = Field(..., ge=1)
+
+
+class OrderCreate(BaseModel):
     order_items: List[OrderItemCreate]
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)